src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -454,4 +466,55 @@ public VisualEditorEditTemplateDialog openEditTemplateDialog() {
     clickContextMenu();
     return new VisualEditorEditTemplateDialog(driver);
   }
+
+  public VisualEditorPageObject clickInsertToolButton() {
+    for (int i=0; i<toolsList.size(); i++) {
+      if (toolsList.get(i).getText().contentEquals("Insert")) {
+        toolsList.get(i).click();
+      }
+    }
+    return this;
+  }
+
+  public VisualEditorPageObject clickInsertInfoboxFromInsertToolMenu() {
+    for (int i=0; i<insertMenuTools.size(); i++) {
```

Comment:
Why is this loop necessary? Don't you just need to click a single element? However, I could be wrong since I'm not 100% sure what is being checked.